### ISSUE TITLE
save the setup

### ISSUE THREAD

**Issue by amaussa** [smouksassi/ggquickeda#12]
possibility to save the setup.
I know seem complex to implement, but help to don't redo to add all parameter all the time.
some idea:
-  do the same concept of the sale plot.
possibility to select before import the .csv in the list
or
place this file  parameters…. closer the .csv file, if this file exist import in the same time of the .csv
better if this config file is editable in ascii
Thanks
Alain

**Comment by smouksassi** [comment]
yes that is a long term goal but for the moment there is no bandwidth will keep this open for visibility

**Comment by smouksassi** [comment]
with latest release we supported in part this request please check it out
